src/modules/backtrace.ts

```diff
@@ -104,8 +104,12 @@ export default class BacktraceHelper {
    * @returns {string[]}
    */
   private getSourceFileAsLines(filepath: string): string[] {
-    return fs.readFileSync(filepath, 'utf-8')
-      .split('\n');
+    try {
```

Comment:
it would be better to check file existance instead of wrapping in a try-catch statement